data-model/src/main/java/io/micronaut/data/model/query/builder/AbstractSqlLikeQueryBuilder.java

```diff
@@ -1437,8 +1438,15 @@ private void appendCriteriaForOperator(StringBuilder whereClause,
             }
 
         } else {
+            // TODO: Expand values for embedded field (composite id).
```

Comment:
So, this is pretty complex issue since we get `List<ProjectId>` where `ProjectId` is embedded/composite id. Then we need to traverse through embedded properties and extract fields from the id, but we can't be sure what is actually in the list. Since this is not frequent use case, maybe we can skip it now, or if there is an approach that works securely please let me know.